core/src/main/java/org/apache/brooklyn/util/core/task/Tasks.java

```diff
@@ -492,4 +494,29 @@ private static String getTimeoutString(Repeater repeater) {
         return "in "+timeout;
     }
 
+
+    public static void dumpInfo(Task<?> t) {
```

Comment:
Minor (and not for this PR): we should probably move this stuff into a separate `Dumper` utility class, rather than `Entities` and `Tasks` being a dumping ground for lots of stuff.